app/helpers/Emails/NotificationsHelper/Comments/SolutionCommentsEmailsSender.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace App\Helpers\Notifications;
+
+use App\Helpers\EmailHelper;
+use App\Model\Entity\AssignmentSolution;
+use App\Model\Entity\Comment;
+use App\Model\Entity\ReferenceExerciseSolution;
+use Latte;
+use Nette\Utils\Arrays;
+
+/**
+ * Sending emails on solution commentary.
+ */
+class SolutionCommentsEmailsSender {
+
+  /** @var EmailHelper */
+  private $emailHelper;
+  /** @var string */
+  private $sender;
+  /** @var string */
+  private $assignmentSolutionCommentPrefix;
+  /** @var string */
+  private $referenceSolutionCommentPrefix;
+
+  /**
+   * Constructor.
+   * @param EmailHelper $emailHelper
+   * @param array $params
+   */
+  public function __construct(EmailHelper $emailHelper, array $params) {
+    $this->emailHelper = $emailHelper;
+    $this->sender = Arrays::get($params, ["emails", "from"], "noreply@recodex.cz");
```

Comment:
The recodex.cz is nonexistent domain. Should not it be prudent to replace recodex.cz with recodex.mff.cuni.cz (or recodex.localhost if you insist on being completely independent of the founding institution).